wsgi/odyssy/announcement/models.py

```diff
@@ -37,13 +37,20 @@ def get_single_announcement(announcement_id):
     def get_latest_announcements(cnt):
         """ get latest announcement """
         return Announcement.objects.filter(
-            Q(initDate__gte=datetime.datetime.today())|
-            Q(finDate__gte=datetime.datetime.today())
+            Q(initDate__gte=datetime.today())|
+            Q(finDate__gte=datetime.today())
             ).order_by('-initDate')[:cnt]
 
+    def get_default_start_date():
```

Comment:
Rename this function to something better suited for its usage. It will give start time for current date, so name something like that. Also, make it `@staticmethod`.

Same applied for the other function.